docs/source/dataframe-joins.rst

```diff
@@ -33,12 +33,17 @@ parallel way, where each partition of the large DataFrame is joined against the
 single small table.  This incurs almost no overhead relative to Pandas joins.
 
 If your smaller table can easily fit in memory, then you might want to ensure
-that it is a single partition with the following
+that it is a single partition with the repartition method.
 
 .. code-block:: python
 
+    from dask.datasets import timeseries
+    large = timeseries(freq="10s", npartitions=10)
+    small = timeseries(freq="1D", dtypes={"z": int})
```

Comment:
This is obviously not part of this work, but on first read it was not at all clear to me that keys of `dtypes` contains the complete set of column names. Do you think it would be clearer if this kwarg was renamed to columns?

Reply:
Yes, I think it would be clearer. Do we need this to be backward compatible or can I just go ahead and rename it?

Reply:
We should go through a deprecation process so that `dtypes` still works for a while, but produces a `FutureWarning`. Here's an example:
https://github.com/dask/dask/blob/95ebb397c328576cb1e6dbbfa6aff2e5b040bac9/dask/array/percentile.py#L119-L125